conf/airframes/examples/bebop.xml

```diff
@@ -10,6 +10,17 @@
       <module name="udp"/>
     </target>
 
+    <define name="MT9F002_OUTPUT_HEIGHT" value="640" />
```

Comment:
Yes I can look into this. What about something like 3 different combinations (1 high fps, 1 wide angle, 1 in between) and put them in the airframe commented out?